akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -1609,6 +1609,37 @@ final class Source[Out, Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[
       matF: function.Function2[Mat, M, M2]): javadsl.Source[Out, M2] =
     new Source(delegate.interleaveMat(that, segmentSize, eagerClose)(combinerToScala(matF)))
 
+  /**
+   * Interleave is a deterministic merge of the given [[Source]] with elements of this [[Flow]].
```

Comment:
This is in `Source` so there is no `Flow`?

Reply:
but I see we have that for other things as well in javadsl.Source. In scaladsl the same FlowOps is used.